docker-compose.yml

```diff
@@ -14,6 +14,7 @@ x-common-django:
     - backup-restore:/backup_restore
     - data:/data
     - tmp:/tmp
+    - logs:/var/log/
```

Comment:
are we sure that having a shared `/var/log` root folder between multiple containers is fine? This is a standard path for many services and applications, beyond GeoNode and Celery. I fear that might race to write to the same positions. Is my suspect wrong?

Reply:
I don't know why this comment was marked as resolved but, to me, it isn't resolved. Have you tested the effects of having a shared `/var/log` between multiple containers @jkariukidev?

Reply:
Yes. In this case, different services are writing to the directory at the same time and would be better to distinguish the services present in GeoNode where they write to. I have tested with `/opt` and the logs are being written in the following fashion. 
![image](https://user-images.githubusercontent.com/23359514/168576930-47bc06b7-22f5-453b-9a79-28e6ce3d22a3.png)